dnf.spec

```diff
@@ -111,11 +111,8 @@ BuildRequires:  python-iniparse
 BuildRequires:  python-libcomps >= %{libcomps_version}
 BuildRequires:  python-librepo >= %{librepo_version}
 BuildRequires:  python-nose
-%if 0%{?rhel} && 0%{?rhel} <= 7
```

Comment:
@ignatenkobrain What about CentOS and RHEL? If I am correct, python2-gpg is not present there.